examples/WritingSALScript.ipynb

```diff
@@ -0,0 +1,348 @@
+{
```

Comment:
<p>Maybe add a sentence saying what <code>await</code>  means?</p><p>"The control software performs numerous tasks asynchronously. Using an <code>await</code>  statement ensures that the command will not return until it's completed. For more information see documentation in the <code>asyncio</code>  library"  (or wherever, you get the idea).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>